src/pages/UserOverview/UserOverview.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {useLocation} from 'react-router-dom';
+import {UserData} from 'types/types';
+import {getUserColumns} from 'utils/utils';
+import Card from '../../components/Card/Card';
+import {GlobalContainer} from '../../components/global.styled';
+import Header from '../../components/Header/Header';
+
+const getUserCard = (user: UserData) => {
```

Comment:
renamed the method from mapU to getUserCard , to better understand what the method does, and specified the returning type